### ISSUE TITLE
django 1.9a compatibility

### ISSUE THREAD

**Issue by bastianh** [edoburu/django-debugtools#2]
Hi,

I know django 1.9 is not yet released, but I'm already peeking and maybe someone has already begun testing and fixing django-debugtools?

When using the new alpha, django won't start with error: `django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.`

Thanks

**Comment by bastianh** [comment]
for everyone looking, the problem is importing 

a solution is described there: http://stackoverflow.com/a/31949071

**Comment by vdboor** [comment]
Wow, thanks for looking into this, and offering a solution already.

Could you turn this into a pull request which updates the code (`if django.VERSION < (1,9)`) and provides the README instructions?

**Comment by bastianh** [comment]
Yeah.. I will check in the next days. There are also some deprecation warnings we can fix.

**Comment by vdboor** [comment]
I've made the change in 3ad0f568b1d9441e8136d522ac1c775a8d80828a, but it needs some testing.

**Comment by bastianh** [comment]
Sorry... I was a bit busy but still had it in my todo list.. I will test your commit and will look after the deprecation warnings.

**Comment by bastianh** [comment]
I have tested your commit with django 1.8.5 and 1.9b1 and both worked. 
I have fixed one deprecation warning and created a pull request.

There is still one deprecation waring in the print_queries template, but I'm not sure how to fix it best to keep it compatible with old versions.
```RemovedInDjango110Warning: The old {% cycle %} syntax with comma-separated arguments is deprecated.
  RemovedInDjango110Warning,```

Maybe adding a new template and load that one depending on version of django?
